editor/editor_node.cpp

```diff
@@ -6476,7 +6486,6 @@ EditorNode::EditorNode() {
 	register_exporters();
 
 	EDITOR_DEF("interface/editor/save_on_focus_loss", false);
-	EDITOR_DEF("interface/editor/show_update_spinner", false);
```

Comment:
Moving this to `editor_settings.cpp` seems to have added it properly to the docs.

So it seems all these properties may be missing from the docs and should be moved to (in a separate PR).